http-clients/src/main/java/com/palantir/remoting/http/FeignClients.java

```diff
@@ -132,14 +101,18 @@ public static FeignClientFactory withMapper(ObjectMapper mapper) {
      * Provides a {@link FeignClientFactory} with the specified {@link ObjectMapper}.
      */
     public static FeignClientFactory withMapper(ObjectMapper mapper, Request.Options timeoutOptions) {
+        return withEncoderAndDecoder(new JacksonEncoder(mapper), new JacksonDecoder(mapper), timeoutOptions);
+    }
+
+    /**
+     * Provides a {@link FeignClientFactory} with the specified {@link Encoder} and {@link Decoder}.
+     */
+    public static FeignClientFactory withEncoderAndDecoder(Encoder encoder, Decoder decoder,
```

Comment:
Freedom to change/refactor it later, because FeignClients is really part of the public part of this library.